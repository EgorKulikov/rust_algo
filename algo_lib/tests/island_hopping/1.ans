2
168.0101570927345
