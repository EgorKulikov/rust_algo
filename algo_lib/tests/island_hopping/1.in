2
3
0.0 0.0
0.0 1.0
1.0 0.0
10
30.0 38.0
43.0 72.0
47.0 46.0
49.0 69.0
52.0 42.0
58.0 17.0
73.0 7.0
84.0 81.0
86.0 75.0
93.0 50.0
