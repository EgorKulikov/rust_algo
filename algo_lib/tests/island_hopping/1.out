2.000
168.01015709273446
